### ISSUE TITLE
Embedded benefit card titles should be <a> tags, not buttons

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1359]

**Comment by brdunfield** [comment]
Resolved by #1379

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility work](https://trello.com/c/H7Q1Ycrm/93-accessibility-work)

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Screenreader Navigation](https://trello.com/c/knKtuM4T/100-accessibility-screenreader-navigation)
